marklogic-data-hub/src/test/java/com/marklogic/hub/deploy/commands/LoadHubModulesCommandTest.java

```diff
@@ -33,9 +33,9 @@
     public void verifyHubVersion() {
         final String message = "Jar version must match version in config.xqy/config.sjs after installation";
 
-        assertEquals(getHubConfig().getJarVersion(), versions.getInstalledVersion(), message);
-        assertEquals(getHubConfig().getJarVersion(), new Versions(getHubClient()).getInstalledVersion(), message);
-        assertEquals(getHubConfig().getJarVersion(), new Versions(getHubConfig()).getInstalledVersion(), message);
+        assertEquals(adminHubConfig.getJarVersion(), versions.getInstalledVersion(), message);
```

Comment:
Same thing for this and the chunk below - need the code on the left, not the right.

Reply:
@rjrudin fixed and pushed, please review.